.github/workflows/model_performance.yml

```diff
@@ -0,0 +1,99 @@
+name: Model Performance Characteristics
+# Schedule triggers
+on:
+  # TODO this should hit after each release.
+  # this is just to make the first few baselines manually
+  pull_request:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
```

Comment:
We should scope down the permissions for the token here
https://github.blog/changelog/2021-04-20-github-actions-control-permissions-for-github_token/

Reply:
this will eventually commit to the repository. Does that mean we'll need `action: write` and that's it?